src/examples/elementary/efl_ui_list_view_example_2.c

```diff
@@ -12,7 +12,7 @@
 #include <Eio.h>
 #include <stdio.h>
 
-#define EFL_MODEL_TEST_FILENAME_PATH "/tmp"
+#define EFL_MODEL_TEST_FILENAME_PATH strdup(eina_environment_tmp_get())
```

Comment:
Hm, should we really force a strdup here?

Reply:
Yep, `eina_environment_tmp_get()` returns a `const char *` and this macro is assigned to a `char *` variable.

Reply:
Oh right, makes sense. Well, and now that I saw that the usage was local to this file, so I think it is okay.